lib/ansible/inventory/manager.py

```diff
@@ -410,6 +410,8 @@ def get_hosts(self, pattern="all", ignore_limits=False, ignore_restrictions=Fals
                 hosts = sorted(self._hosts_patterns_cache[pattern_hash][:], key=attrgetter('name'), reverse=(order == 'reverse_sorted'))
             elif order == 'reverse_inventory':
                 hosts = self._hosts_patterns_cache[pattern_hash][::-1]
+            elif order == 'order_last_dash':
+                hosts = sorted(self._hosts_patterns_cache[pattern_hash][:], key=lambda x: getattr(x, 'name').rsplit("-",1)[-1])
```

Comment:
Looks good -- maar is dit nodig  `..[pattern_hash][:]` ? Of kan dat ook `...[pattern_hash]` worden?